opentelemetry-collector/values.yaml

```diff
@@ -83,6 +83,33 @@ service:
   type: ClusterIP
   annotations: {}
 
+ports: # configures both service, and deployment
+  otlp:
+    enabled: true
+    containerPort: 55681 # new port could also be 4317
```

Comment:
I believe this should be `4317` for GRPC.  We should also add the legacyGRPC port of `55680` since many are still using it, and the collector will create the legacy port on startup if OTLP GRPC is configured.
https://github.com/open-telemetry/opentelemetry-collector/blob/main/receiver/otlpreceiver/otlp.go#L130

Reply:
do you want to own trying to maintain all the port mappings? the struct in the values file can be extended arbitrarily, so users can simply create / append new blocks as needed given it's a `range` over the top level and maps it out by `$key`

Reply:
Right now, this PR will break existing setups that expect port 55680 to be opened by default for OTLP GRPC. So most certainly we need to add the legacyGRPC port.

Since OpenTelemetry is moving to 4317 for GRPC, we should adopt that here as well to be forward compatible.